style.css

```diff
@@ -0,0 +1,920 @@
+  background-image: url(./images/aboutme-bg.png);
+  background-repeat: no-repeat;
+  background-position: right center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.about-me-box {
+  display: flex;
+  flex-direction: column;
+  margin: 10rem 8rem;
+  width: 32%;
+}
+
+.about {
+  font-family: "Crete Round", sans-serif;
+  font-size: 72px;
+  font-weight: 400;
+  line-height: 82px;
+  color: #f55800;
+}
+
+.about-me-note {
+  color: #f4f5f7;
+  font-family: "Inter", sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 32px;
+  margin: 10px 0 10px 0;
+}
+
+.get-my-resume {
+  font-family: "Inter", sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  width: 35%;
+  background-color: #ff6b00;
+  border: none;
+  color: #f4f5f7;
+  padding: 10px 0;
+  margin: 20px 0;
+}
+
+.get-my-resume:enabled {
+  color: #fff;
+  background-color: #f55800;
+}
+
+.get-my-resume:disabled {
+  color: #5b5957;
+  background-color: #f4f1f0;
+}
+
+.get-my-resume:hover:enabled {
+  color: #f55800;
+  background-color: #fff5e1;
+  border: 2px solid #f55800;
+}
+
+.get-my-resume:active:enabled {
+  background-color: #f55800;
+  color: #fff;
+}
+
+@media (max-width: 768px) {
+  .about-me {
+    width: 100%;
+    margin-top: 5rem;
+  }
+
+  .about--me {
+    background-image: url(./images/Frame.png);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    padding-left: 2rem;
+    padding-right: 2rem;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    height: 100vh;
+  }
+
+  .about-me-box {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    width: 100%;
+    height: 30%;
+    margin: 3.5rem 0;
+  }
+
+  .about {
+    font-size: 50px;
+  }
+
+  .about-me-note {
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .get-my-resume {
+    padding: 10px 10px;
+    width: 50%;
+  }
+}
+
+.stack {
+  margin-bottom: 5rem;
+}
+
+.stacks {
+  display: flex;
+  justify-content: space-between;
+  color: #f4f5f7;
+  margin-left: 10%;
+  width: 80%;
+  font-family: "Inter", sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+}
+
+.hr-stack {
+  margin-bottom: 10rem;
+  margin-left: 8rem;
+  margin-right: 8rem;
+}
+
+@media (max-width: 768px) {
+  .stacks {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .hr-stack {
+    margin-left: 2rem;
+    margin-right: 2rem;
+  }
+}
+
+.list {
+  list-style: none;
+  font-family: "Inter", sans-serif;
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 43px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+@media (max-width: 768px) {
+  .list {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-items: center;
+  }
+
+  li.skills {
+    text-align: center;
+  }
+}
+
+li.languages {
+  list-style: none;
+}
+
+li.languages::before {
+  color: orange;
```

Comment:
- [ ] Kindly consider sticking to the Figma design as much as possible by giving these icons the same color as the one given in the design the reason is to make your project an exact duplicate of the given one check the screenshots below to see the difference 
- your design 👇 
<img width="1262" alt="Screen Shot 2023-05-12 at 13 50 22" src="https://github.com/H-Ramalan/portfolio-desktop/assets/60146030/4a5c117e-c01f-45cf-9eda-c4c338ba7dee">



- Figma design 👇 
<img width="704" alt="Screen Shot 2023-05-12 at 13 50 47" src="https://github.com/H-Ramalan/portfolio-desktop/assets/60146030/db6ddec9-5e41-4149-8c5a-de5b489ad4fa">